README.md

```diff
@@ -120,6 +121,22 @@ Previews of links will not be shown if:
   - asked not to show a preview when prompted
   - the GitHub app is not in the channel, which you can remedy with `/invite @github`
 
+#### Mentions
+When you subscribe to a repository in Slack, you will now see yourself mentioned in the notifications where you are referred and needs your attention.
+<p align="left"><img width="500" alt="PR Mentions" src="docs/images/PRMentions.png"></p>
+<p align="left"><img width="500" alt="Deployment Mentions" src="docs/images/DeploymentMentions.png"></p>
+
+When you receive notifications for Issues, PRs and Deployments, here are the cases when you will be mentioned.
+- Assignee in an Issue
+- Reviewer for a PR
+- Mentioned in a PR/Issue description/comment/discussion
+- Reviewer for a Deployment
+- Scheduled reminders for PR review requests
```

Comment:
please tell us, how can we disable it too for a GH person...because for a big team, these notifications will create lots of SLACK TAGGING which means lots of unnecessary distractions 🥺